docs/content.zh/docs/deployment/resource-providers/native_kubernetes.md

```diff
@@ -324,4 +324,209 @@ $ kubectl create clusterrolebinding flink-role-binding-flink --clusterrole=edit
 
 Please refer to the official Kubernetes documentation on [RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) for more information.
 
+### Pod Template
+
+Flink allows users to define the JobManager and TaskManager via pod template file with the advanced features(e.g. volumes, init container, sidecar container, etc.),
+which are not supported by Flink [Kubernetes config options]({{< ref "docs/deployment/config" >}}#kubernetes) directly.
+Use [kubernetes.pod-template-file]({{< ref "docs/deployment/config" >}}#kubernetes-pod-template-file)
+to specify a local file that contains pod definition. It will be used to initialize the
+JobManager and TaskManager instead of an empty pod. Please note that some fields of the pod template
+will always be overridden by Flink.
+Refer to the following tables for the full list of pod fields that will be overwritten.
+All the fields defined in the pod template and not listed in the tables will be unaffected.
+
+Please note that some new config options may be introduced to overwrite more Kubernetes fields,
+your pod template may not work in the future Flink releases.
+We could guarantee that `annotations`, `labels`, `imagePullSecrets`, `nodeSelector`, `tolerations`,
+`env`, `init container`, `sidecar container`, `volumes` from pod template will always be respected.
+
+#### Pod Metadata
+<table class="table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Pod MetaData Key</th>
+            <th class="text-left" style="width: 30%">Modified Value</th>
+            <th class="text-left" style="width: 50%">Description</th>
```

Comment:
Looking at the table, it seems the fields Flink overwrite can be categorized as follows.
* **Defined by Flink.** Definitions in template won't take effect. Use cannot control the fields
* **Defined by Config Option.** Definitions in template won't take effect. User can control the fields via config options.
* **Overwritten by Flink.** Definitions in template will take effect, with additional contents from Flink.

I would suggest to explain these categories in advance, and make it a column in the tables. Thus, the tables may contain the following columns: key, category, related config option, description. In this way, the first three columns might be more concise and easier to understand, while the last column provides details that may not be necessary but good to know.